packages/quantic/force-app/main/default/lwc/quanticCitation/quanticCitation.js

```diff
@@ -112,29 +121,32 @@ export default class QuanticCitation extends NavigationMixin(LightningElement) {
     this.removeBindings?.();
     clearTimeout(this.timeout);
     this.hideTooltipDebounced?.cancel();
+    this.showTooltipDebounced?.cancel();
   }
 
   handleCitationMouseEnter() {
-    this.showTooltip();
+    this.hideTooltipDebounced.cancel();
+    this.showTooltipDebounced();
   }
 
   handleCitationMouseLeave() {
+    this.showTooltipDebounced.cancel();
     this.hideTooltipDebounced();
   }
 
+  /**
+   * Handles mouse enter on the tooltip to prevent it from hiding while hovering over it.
+   */
   handleTooltipMouseEnter() {
+    this.showTooltipDebounced.cancel();
```

Comment:
Why is this needed? why should a mouse enter in the tooltip cancel a `showtooltip` ?